_tools/types/main.go

```diff
@@ -27,21 +27,23 @@ const (
 	fixtureDir = "fixtures"
 	fixtureExt = ".sem.uast"
 	goDocURL   = "https://godoc.org/github.com/bblfsh/sdk/uast#"
+	pprofAddr  = "localhost:6060"
 )
 
 var (
 	reposRootPath = filepath.Join(".", "drivers")
 	uastNodeRe    = regexp.MustCompile("uast:[^\"]*")
 
-	pprof      = flag.Bool("pprof", false, "start pprof profiler http endpoing")
+	pprof = flag.Bool("pprof", false,
+		fmt.Sprintf("Start a pprof profiler HTTP service at %s", pprofAddr),
```

Comment:
Thanks!

Or simpler, if you like: `flag.Bool("pprof", false, "Start a pprof profiler HTTP service at "+pprofAddr)`

(this is fine too)

Reply:
There is a single right way in Go for everything, but a string interpolation 🤷‍♂ 
Thank you for simplification suggestion! History re-writtne in 58dc92b